.cargo/config

```diff
@@ -1,5 +1,8 @@
-[target.thumbv7em-none-eabihf]
-runner = 'arm-none-eabi-gdb'
+[target.thumbv7m-none-eabi]
+
```

Comment:
This one too, please. ;)

Reply:
My bad, thought it made sense to leave it. Removed now.